### PR TITLE
Fix #23: add dlls to jar file and extract if needed at runtime to the user's l…

### LINKED ISSUE #23: Load DLL's from inside JAR
The code failed to load the dll from jar because the jar file on maven central does not contain the dll.

### PR DESCRIPTION
**Pull request by jjYBdx4IL**
…ocal config directory

Also moved the Tester program to the main sources and renamed it to *Demo so one is able to run it directly from the produced jar.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -133,28 +136,6 @@
...
-				<version>1.4</version>
-				<executions>
-					<execution>
-						<id>sign-artifacts</id>
-						<phase>verify</phase>
-						<goals>
-							<goal>sign</goal>
-						</goals>
-					</execution>
-				</executions>
```
Comment (melloware) [inline pom.xml]: You removed the GPG plugin but I need this to do Maven Central Releases.

**Comment by jjYBdx4IL** [comment]
I just moved it to the release profile. The gpg plugin shouldn't be executed when running a simple "mvn clean install".

**Comment by melloware** [comment]
Great PR. Thanks!   I wonder if I should cut a release to Maven Central with this PR.  We still need to figure out the JDK 11 issues but might be worth putting this out there?

**Comment by jjYBdx4IL** [comment]
Yeah I would wait for the Java 11 stuff to be fixed before release.

**Pull request merged**